### PR TITLE
HifiClone API

### PR DESCRIPTION
**Pull request by boli-bohan**
- wrapper around `/voices/clone/hifi` endpoint

### CODE REVIEW

File: src/voices/index.ts [hunk 1]
```diff
@@ -56,6 +57,27 @@ export default class Voices extends Client {
...
+	async createHifiClone(options: HifiCloneOptions): Promise<Voice> {
+		const formData = new FormData();
+		formData.append("clip", options.clip);
+		formData.append("name", options.name);
+		formData.append("description", options.description);
+		formData.append("language", options.language);
+		formData.append("model_id", options.model_id);
+		if (options.transcript) {
+			formData.append("transcript", options.transcript);
+		}
```
Comment (kbrgl) [inline src/voices/index.ts:69]: Isn't this required for Hi-Fi cloning? Why is the user allowed to omit it?

**Review by kbrgl — APPROVED** [review]

File: src/voices/index.ts [hunk 2]
```diff
@@ -56,6 +57,27 @@ export default class Voices extends Client {
 		throw new Error("Invalid mode for clone()");
 	}
 
+	async createHifiClone(options: HifiCloneOptions): Promise<Voice> {
```
Comment (kbrgl) [inline src/voices/index.ts:60]:
```suggestion
	async hifiClone(options: HifiCloneOptions): Promise<Voice> {
```

cleaner

**Review by kbrgl — APPROVED** [review]

File: src/voices/index.ts (hunk 1 above)
Comment (boli-bohan) [reply src/voices/index.ts:69]: We'll generate it for them via whisper if they omit it.

File: src/types/index.ts [hunk 3]
```diff
@@ -131,6 +131,16 @@ export type CloneOptions =
 			enhance?: boolean;
 	  };
 
+export type HifiCloneOptions = {
+	clone_options: CloneOptions;
```
Comment (noahlt) [inline src/types/index.ts:135]: preference: don't nest this

```
export type HifiCloneOptions = CloneOptions & {
  ...
};

File: src/types/index.ts [hunk 4]
```diff
@@ -131,6 +131,16 @@ export type CloneOptions =
...
+	name: string;
+	description: string;
```
Comment (noahlt) [inline src/types/index.ts:137]: shouldn't description be optional?

Comment (boli-bohan) [reply src/types/index.ts:137]: I want it to be empty string instead of null

**Review by noahlt — APPROVED** [review]

**Pull request merged**